rhcos-layer/README.md

```diff
@@ -0,0 +1,88 @@
+# Creating RHCOS image Layer
+
+This folder contains artifacts to create RHCOS image layer for SNP and TDX.
+
+The RHCOS image layer contains the required kernel version, kata and qemu rpms.
+
+## Download the required rpms
+
+For SNP
+
+```sh
+cd rpms-download
+./download-rpms.sh rhel-rpms.yaml
+```
+
+Copy the downloaded rpms to the `snp/rpms` folder
+
+For TDX
+
+```sh
+cd rpms-download
+./download-rpms.sh centos-rpms.yaml
+```
+
+Copy the downloaded rpms to the `tdx/rpms` folder
+
```

Comment:
We also need to mention how to get the kata-containers rpm under snp/rpms and tdx/rpms.

Reply:
I'll add the Containerfile for kata-containers rpm. Wondering how to point to the src rpm. 
Given it's size, I don't know if it makes sense to version it here. Any thoughts ?

Reply:
What about creating a docker/podman image with a version (e.g. 0.2.0.<date>) which could contain kata-container rpm and srpm ?

Reply:
I see.. I do have the rpms in a container image for now - quay.io/bpradipt/rhcos-layer/rpms:0.2.0. Plan to move the final one to https://quay.io/openshift_sandboxed_containers along with other artefacts